control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs

```diff
@@ -24,13 +24,20 @@ impl crate::controller::io_engine::HostApi for super::RpcClient {
                 request: "v1::get_mayastor_info",
             })?;
 
-        let registration_info = match data.into_inner().registration_info {
+        let inner = data.into_inner();
+
+        let registration_info = match inner.registration_info {
```

Comment:
nit: preferred it as a single line :D

Reply:
That was causing issue below in line 39, returning the version.